source/credit/index.rst

```diff
@@ -1,58 +1,170 @@
+-------------------------
 
-Requirements for latest version
--------------------------------
+============ =========== ===========
+GLPI Version Minimum PHP Recommanded
+============ =========== ===========
+10.0.x       8.1         8.2
+============ =========== ===========
 
-This plugin requires :
+Install the plugin
+------------------
 
-* PHP 7.2 or higher
-* GLPI >= 9.4
+.. note::
+   This plugin is community and does not need a license. It is also available in Cloud.
 
+-  Go to the marketplace. Download and install the plugin “**Credit**”.
 
-Features
---------
-This plugin allows you to declare and follow (by entity) the consumptions of different credit vouchers directly from the Ticket form.
+.. figure:: images/Credit-1.png
+   :alt:
 
-To do it, you must follow these steps:
+Setting up the plugin
+---------------------
 
-* Creation of different types of credit vouchers (as GLPI dropdown);
-* Creation, in Entity tab, of the credit vouchers available for these entities;
-* Declaration / Modification / Deletion of a consumption in the Ticket form and tab;
-* Consultation of the vouchers sold, the remaining credits, credits consumptions from the Ticket form or from Entity tab.
+Creating a credit type
+~~~~~~~~~~~~~~~~~~~~~~
 
-Configuration
--------------
+-  From **setup > dropdowns**,
+-  click on **credits** then **credit types**.
 
-The plugin will create required tables in the database  automatically. Those tables will be updated along with the plugin.
+.. figure:: images/Credit-2.png
+   :alt:
 
-Plugin usage is quite simple:
+-  Click on **add**.
+-  Enter a name representing the type of credit granted
 
+.. figure:: images/Credit-3.png
+   :alt:
 
-You will access the credit configuration from the *Administration* > *Entities*.
+Add credit to customer entity(ies)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. image:: images/create.png
 
-* Name : add name to credit
-* Type : Define a type
-* Start / End date : Define duration of credit
-* Active : Display credit on ticket form or not
-* Quantity : quantity spend
-* Allow overconsumption : allow or not over consumptin (if yes, check is bypass)
+Example : “Case study for this article” We will create 3 types of credit:
 
-Credit consumption
-------------------
+   - For a new follow-up
+   - For a new task
+   - For a new solution
+
+   You can create a single type of credit which will group together the 3, it depends on the management of each case.
+
+-  In **administration > entities**, select the entity that will receive the credits.
+-  Go to the **credit** section
+-  In the section **add a credit**, enter a name for the new credit
+-  Define the type of credit created earlier
+-  The start and end dates of the credit
+-  Activate it,
+-  Make it available or not in the child entities (note that if you are in the parent entity, it will be accessible to all entities)
+-  The quantity of credit sold
+-  The authorisation to overconsume or not
+-  Click on **add**
```

Comment:
```suggestion
-  In **Administration > Entities**, select the entity that will receive the credits.
-  Go to the **Credit** section
-  In the section **Add a credit**, enter a name for the new credit
-  Define the type of credit created earlier
-  The start and end dates of the credit
-  Activate it,
-  Make it available or not in the child entities (note that if you are in the parent entity, it will be accessible to all entities)
-  The quantity of credit sold
-  The authorisation to overconsume or not
-  Click on **add**
```